Forms/AHKForm.cs

```diff
@@ -62,13 +69,12 @@ private void txtSpammerDelay_TextChanged(object sender, EventArgs e)
         {
             try
             {
-                ProfileSingleton.GetCurrent().AHK.ahkDelay = Int16.Parse(this.txtSpammerDelay.Text);
+                ProfileSingleton.GetCurrent().AHK.AhkDelay = Convert.ToInt16(this.txtSpammerDelay.Value);
```

Comment:
Fix a bug where we save the previous value and not the current value configured in the `UpDown` input.